data/mk_folders.sh

```diff
@@ -0,0 +1,7 @@
+#!/bin/bash
```

Comment:
Right on, that makes total sense. I can move it to the Makefile and adjust that there.